packages/opc-styles/functions/_custom-properties.scss

```diff
@@ -0,0 +1,109 @@
+/// Author: https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-sass/functions/_custom-properties.scss
+/// Returns the css variable with category type and fallback if provided
+///
+/// this function is called in the opc-color function.
+///
+/// @param {String} $category - Category name to be appended to variables within the map
+/// @param {String} $key - Variable name passed by opc-color function
+/// @param {Map} $map - SASS Map of variables
+/// @param {String} $fallback [""] - Optional fallback override for custom property
+/// @requires {String} $repo - Global variable definition
+/// @requires {Boolean} $USE-FALLBACK - True to use fallback colors, false to omit
+///
+@function opc-get-from-map($category, $key, $map, $fallback: null) {
```

Comment:
@gisumit can you mention what does this function do? why it is named as opc-get-from-map?

Reply:
yeah, it's defined.